sample-apps/python/django_frontend_service/frontend_service_app/views.py

```diff
@@ -105,3 +107,26 @@ def get_xray_trace_id():
     xray_trace_id = f"1-{trace_id[:8]}-{trace_id[8:]}"
 
     return JsonResponse({"traceId": xray_trace_id})
+
+def mysql(request):
+    logger.info("mysql received")
+    cur = None
+    conn = None
+    try:
+        conn = pymysql.connect(
+            host=os.environ["RDS_MYSQL_CLUSTER_ENDPOINT"],
+            user=os.environ["RDS_MYSQL_CLUSTER_USERNAME"],
+            password=os.environ["RDS_MYSQL_CLUSTER_PASSWORD"],
+            database=os.environ["RDS_MYSQL_CLUSTER_DATABASE"]
+        )
+        cur = conn.cursor()
+        cur.execute("SELECT * FROM tables LIMIT 1;")
+        return get_xray_trace_id()
+    except Exception as exception:  # pylint: disable=broad-except
+        logger.exception("Exception Occurred")
+        raise exception
+    finally:
+        if cur:
+            cur.close()
+        if conn:
+            conn.close()
```

Comment:
Nitpick: Could we use Python `with-resource-open` feature to let it close connections automatically for us?
See: https://github.com/PyMySQL/PyMySQL

Reply:
Yes, will address this in the next commit